### ISSUE TITLE
Process individual Granules

### ISSUE THREAD

**Issue by benboughton1** [umwilm/SEN2COR#4]
It would be great to be able to run sen2cor on individual granules. E.g.

L2_Process --resolution 10 S2A_OPER_MSI_L1C_TL_SGS__20160101T063159_A002744_T55JGH_N02.01

Any plans for such functionality?

**Comment by umwilm** [comment]
yes, there is a new release in preparation which will do a parallelization of tile processing. With this it will also be possible to process tiles individually. The version will be available soon.

**Comment by benboughton1** [comment]
Excellent news. Thanks for your hard work.

**Comment by benboughton1** [comment]
When can we expect next release? Thanks

**Comment by umwilm** [comment]
Within the next two weeks.
Cheers,
Uwe

**Comment by benboughton1** [comment]
Have there been some delays in the latest release?

**Comment by umwilm** [comment]
We are in the final tests …

**Comment by benboughton1** [comment]
Can we install 2.1.x from master now?

**Comment by umwilm** [comment]
Hi Ben,

it would not make much sense to download from this repository. The atmospheric correction module of sen2cor which is –the- essential part of the software contains an IPR of the Deutsche Luft und Raumfahrt, and is not allowed to be published. Thus will never be part of this repository. Consequently, the software downloaded from here will definitively not work. This is only a private repository, thought to be for interchange between the sen2cor developers. The official way to download the software (including the module for the atmospheric correction) is from http://s2tbx.telespazio-vega.de/sen2cor/. The software (release 2.1.X) will be published there, after getting the official clearance from side of ESA.

Thus, I have to ask for your patience until this clearance has been received.

Thank you for your understanding,
Cheers,
Uwe

**Comment by benboughton1** [comment]
Ok no worries - I now understand and will wait for official release. Thanks


Sent from Outlook Mobile






On Fri, Mar 11, 2016 at 1:45 AM -0800, "Uwe Müller-Wilm" <notifications@github.com> wrote:










Hi Ben,





it would not make much sense to download from this repository. The atmospheric correction module of sen2cor which is –the- essential part of the software contains an IPR of the Deutsche Luft und Raumfahrt, and is not allowed to be published. Thus will never be part of this repository. Consequently, the software downloaded from here will definitively not work. This is only a private repository, thought to be for interchange between the sen2cor developers. The official way to download the software (including the module for the atmospheric correction) is from http://s2tbx.telespazio-vega.de/sen2cor/. The software (release 2.1.X) will be published there, after getting the official clearance from side of ESA.





Thus, I have to ask for your patience until this clearance has been received.





Thank you for your understanding,


Cheers,


Uwe
